doc/news/changes/minor/20190306BrunoTurcksin

```diff
@@ -0,0 +1,4 @@
+Fixed: LinearAlgebra::CUDAWrappes::Vector::add_and_dot was giving a wrong result
+if the size of vector was greater than 4096
```

Comment:
CUDAWrappes -> CUDAWrappers

Also, the sentence should be terminated with a period.